### DISCUSSION TITLE
Server banner

### DISCUSSION THREAD

**Discussion by Supersimmons** [romanz/electrs#829] [General]
Hello,
is it possibile to modify welcome server banner?
For example:
Server Banner: Welcome to electrs 0.9.11 (Electrum Rust Server)!

to

Server Banner: Welcome to my electrs server!

Thanks

**Comment by romanz** [comment]
Please use the `server_banner` config:
https://github.com/romanz/electrs/blob/172d76bbad378da8d3ca3b88c4f17e2473dec34e/internal/config_specification.toml#L125-L128
